awx/main/credential_plugins/aim.py

```diff
@@ -0,0 +1,83 @@
+from .plugin import CredentialPlugin
+
+import os
+import stat
+import tempfile
+import threading
+from urllib.parse import urljoin
+
+from django.utils.translation import ugettext_lazy as _
+import requests
+
+
+aim_inputs = {
+    'fields': [{
+        'id': 'url',
+        'label': _('AIM URL'),
```

Comment:
Not to be confused with "AOL Instant Messager 😂 "
Maybe CyberArk AIM URL?

Reply:
😂good point - fixed